### PR TITLE
chore(testing): make tests slightly more readable

### PR DESCRIPTION
**Pull request by emmettbutler**
This pull request contains changes pulled out of https://github.com/DataDog/dd-trace-py/pull/4863. They're thematically unified in that they make little adjustments to tests that are affected by module cloning. More detailed motivation for each change can be found in the commit messages.

## Checklist

- [x] Change(s) are motivated and described in the PR description.
- [x] Testing strategy is described if automated tests are not included in the PR.
- [x] Risk is outlined (performance impact, potential for breakage, maintainability, etc).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) are followed.
- [x] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
- [x] Author is aware of the performance implications of this PR as reported in the benchmarks PR comment.

## Reviewer Checklist

- [x] Title is accurate.
- [x] No unnecessary changes are introduced.
- [x] Description motivates each change.
- [x] Avoids breaking [API](https://ddtrace.readthedocs.io/en/stable/versioning.html#interfaces) changes unless absolutely necessary.
- [x] Testing strategy adequately addresses listed risk(s).
- [x] Change is maintainable (easy to change, telemetry, documentation).
- [x] Release note makes sense to a user of the library.
- [x] Reviewer is aware of, and discussed the performance implications of this PR as reported in the benchmarks PR comment.

### CODE REVIEW

File: tests/contrib/logging/test_tracer_logging.py [hunk 1]
```diff
@@ -202,9 +202,9 @@ def test_logs_with_basicConfig(run_python_code_in_subprocess, ddtrace_run_python
 
         code = """
 import logging
-import ddtrace
```
Comment (Yun-Kim) [inline tests/contrib/logging/test_tracer_logging.py:205]: Why were we able to remove this? Can you help me understand why removing this line doesn't result in line 208/210 messing up below?
```python
ddtrace_logger = logging.getLogger('ddtrace')
```

**Review by Yun-Kim — COMMENTED** [review]
LGTM, just one small question

File: tests/contrib/logging/test_tracer_logging.py (hunk 1 above)
Comment (emmettbutler) [reply tests/contrib/logging/test_tracer_logging.py:205]: That was an oversight on my part. Thanks for catching it.

**Review by Yun-Kim — DISMISSED** [review]

File: tests/contrib/logging/test_tracer_logging.py [hunk 2]
```diff
@@ -231,23 +232,9 @@ def test_warn_logs_can_go_to_file(run_python_code_in_subprocess, ddtrace_run_pyt
...
-assert isinstance(ddtrace_logger.handlers[0], logging.handlers.RotatingFileHandler)
-assert ddtrace_logger.handlers[0].maxBytes == 200000
-assert ddtrace_logger.handlers[0].backupCount == 1
-
-ddtrace_logger.warning('warning log')
-"""
-
-    ddtrace_run_code = """
-import logging
-
```
Comment (Yun-Kim) [inline tests/contrib/logging/test_tracer_logging.py:250]: I'm not sure if we can actually merge the `patch_code` and `ddtrace_run_code` into one code block. I thought the point of this test is to test the different execution methods of code (import ddtrace, or ddtrace-run) and make sure they're the same, but by importing ddtrace in the `ddtrace-run` version of the code doesn't that defeat the purpose?

**Review by Yun-Kim — COMMENTED** [review]
On second thought I don't know if we can justify pulling these changes out of #4863, since aside from the patch code change (which I'm not sure if we should even do) these are just minor edits to make the tests slightly more readable. WDYT?

**Comment by emmettbutler** [comment]
@Yun-Kim my view is "why not pull these changes out?" It creates an independently mergeable pull request and makes the original easier to review, without any apparent downside.

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**